docker/README.md

```diff
@@ -28,6 +28,12 @@ $ git clone git@github.com:openfoodfoundation/openfoodnetwork.git
 ```sh
 $ cd openfoodnetwork
 ```
+* If this is your first time using docker, make sure docker-compose is installed:
+
+```sh
+$ sudo apt install docker-compose
```

Comment:
I now see that we mention docker-compose in line 11 already. I'm wondering if that's easily overlooked. What would have helped you to realise you needed docker-compose? Should these commands be up there? Maybe in a `<details>` tag? Or should the headline say: "Requirements - installing Docker"? Or maybe it needs different formatting up there?

Just some ideas. It probably needs improving but the essential information is already there. But maybe it just sounds like there's additional information instead of telling you that you *need* to do that.